### PR TITLE
Initial setup for micromasters portal

### LINKED ISSUE #9: Set up initial app
start with cookie cutter app

### PR DESCRIPTION
**Pull request by noisecapella**
Fixes #9

### CODE REVIEW

File: micromasters/settings.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+INSTALLED_APPS = (
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    # Our INSTALLED_APPS
+    'ui'
```
Comment (noisecapella) [inline micromasters/settings.py]: We would have that problem with any name, wouldn't it? I'm not saying that all ui has to go within this app, it's just a place to start writing code

File: ui/__init__.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+"""ui
```
Comment (noisecapella) [inline ui/__init__.py]: It's from the cookiecutter, I'll remove it

File: ui/__init__.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+Web Portal for MicroMasters
+"""
+
+__project__ = 'ui'
+__version__ = '0.0.0'
```
Comment (noisecapella) [inline ui/__init__.py]: It might be best to just get rid of this and make it blank

**Comment by bdero** [comment]
#24 is live at https://micromasters.herokuapp.com/, which is a child PR of this PR. :+1:

**Pull request merged**